ballerina-to-openapi/src/main/java/io/ballerina/openapi/service/mapper/OpenAPIResourceMapper.java

```diff
@@ -287,9 +301,10 @@ private String generateRelativePath(FunctionDefinitionNode resource) {
         relativePath.append("/");
         if (!resource.relativeResourcePath().isEmpty()) {
             for (Node node: resource.relativeResourcePath()) {
-                if (node instanceof ResourcePathParameterNode pathNode) {
+                if (node instanceof ResourcePathParameterNode) {
+                    ResourcePathParameterNode pathNode = (ResourcePathParameterNode) node;
```

Comment:
Any reason for this change?

Reply:
I removed the casting in here it is not necessary to have it there.. thanks for pointing out :)